lobby-db-dao/src/main/java/org/triplea/lobby/server/db/dao/BadWordsDao.java

```diff
@@ -17,6 +17,9 @@
   @SqlUpdate("delete from bad_word where word = :word")
   int removeBadWord(@Bind("word") String badWordToRemove);
 
+  /*  @SqlQuery("select exists (select * from game_hosting_api_key where key = :apiKey)")
+  boolean keyExists(@Bind("apiKey") String apiKey);*/
+  // TODO: change the count to an exists
```

Comment:
Obligatory TODO warning, also there's a comment containing another query

Reply:
Ah, i was mostly kicking this PR out the door last night before heading to bed, meant to put this file to another branch. Thinking I'll remove it from this PR.